lib/ecto/migration.ex

```diff
@@ -1035,6 +1046,10 @@ defmodule Ecto.Migration do
     * `:comment` - adds a comment to the added column.
     * `:after` - positions field after the specified one. Only supported on MySQL,
       it is ignored by other databases.
+    * `:start_value` - option for `:identity` key, represents initial value in sequence
+      generation. Defaults to `0`.
+    * `:increment` - option for `:identity` key, represents increment value for
+      sequence generation. Defaults to `1`.
```

Comment:
```suggestion
    * `:start_value` - option for `:identity` key, represents initial value in sequence
      generation. Defaults is defined by the database.
    * `:increment` - option for `:identity` key, represents increment value for
      sequence generation. Default is defined by the database.
```

Since we are not setting the defaults in Ecto? 

https://github.com/elixir-ecto/ecto_sql/blob/master/lib/ecto/adapters/postgres/connection.ex#L1254